### ISSUE TITLE
pip install fails

### ISSUE THREAD

**Issue by tobyaharris** [DigitalShoestringSolutions/InventoryManagement#12]
Tests on stable and development this morning fail at `docker compose build`. 
```
 => ERROR [web 4/7] RUN pip install --upgrade pip && pip install -r requirements.txt                                                                                                                                                     0.4s
------                                                                                                                                                                                                                                        
 > [web 4/7] RUN pip install --upgrade pip && pip install -r requirements.txt:
0.294 /bin/sh: error while loading shared libraries: libc.so.6: ELF load command address/offset not page-aligned
------
failed to solve: process "/bin/sh -c pip install --upgrade pip && pip install -r requirements.txt" did not complete successfully: exit code: 127
```
Perhaps the [requirements versioning issue](https://github.com/DigitalShoestringSolutions/Inventory_management/issues/11) is relevant

**Comment by tobyaharris** [comment]
In quick testing none of the below fix this issue:
- Removing the unsecured dependency (`openpyxl`) , 
- Removing the dockerfile command to upgrade pip (reducing `RUN pip install --upgrade pip && pip install -r requirements.txt` to `RUN pip install -r requirements.txt`)
- specifing pip3 (changing said dockerfile line to `RUN pip3 install -r requirements.txt`. With that done, this line is the same as in [Temperature dockerfile](https://github.com/DigitalShoestringSolutions/TemperatureMonitoring/blob/666211f577a197673641bc9525dc01bb91ccaca1/temperature_dc/Dockerfile#L26) which works reliably, including on the same test pi (5) this morning.

**Comment by tobyaharris** [comment]
It's because the dockerfile specifies arm32 (bit), I'm working on a 64bit system. 
I propose using the base image `python:3.8`  which will adapt to the hardware used.

**Comment by tobyaharris** [comment]
Fixed in PR#13
